README.md

```diff
@@ -36,6 +36,12 @@ Similar to Task 2, start the data feed server by running the python server
 
 <code> python2 datafeed/server.py </code>
```

Comment:
@andrew-insidesherpa same as my comment in your module2 pull request, change this to just python, not python2